typescript-test-samples/schema-and-contract-testing/tests/contract-testing.test.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import event_1_0_0 from '../events/event-1.0.0.json';
+import event_1_2_0 from '../events/event-1.2.0.json';
+import event_1_3_0 from '../events/event-1.3.0.json';
+
+import { isAddressValid } from '../src/address-validation';
+
+describe('Conract testing examples', () => {
```

Comment:
Spelling error for `describe` text (I think?)

Reply:
Good catch! Fixed.